Content.Shared/SS220/DarkReaper/DarkReaperSharedSystem.cs

```diff
@@ -370,6 +400,7 @@ public virtual void ChangeForm(EntityUid uid, DarkReaperComponent comp, bool isM
 
         if (isMaterial)
         {
+            EnsureComp<PullerComponent>(uid);
```

Comment:
Вроде как то так
```suggestion
            EnsureComp<PullerComponent>(uid).NeedHands = false;
```
Если будет писать что нет доступа к этому параметру, то добавь к нему аттрибут `[Access(Other = AccessPermissions.ReadWriteExecute)]`